pkg/grpc/exec/exec.go

```diff
@@ -63,6 +64,41 @@ func GetProcessExec(proc *process.ProcessInternal) *tetragon.ProcessExec {
 	}
 }
 
+type MsgCgroupEventUnix struct {
+	processapi.MsgCgroupEvent
+}
+
+func (msg *MsgCgroupEventUnix) RetryInternal(ev notify.Event, timestamp uint64) (*process.ProcessInternal, error) {
+	return nil, fmt.Errorf("Unreachable state: MsgCgroupEventUnix RetryInternal() not implemented")
```

Comment:
I think returning an error here is fine because this event is not going to be in the cache. But maybe we should change the error to something like:
> fmt.Errorf("Unreachable state: MsgCgroupEventUnix: RetryIntenal() was called")

Reply:
Done, thank you @kkourt